### PR TITLE
API GW V2 - Update backend actions

### PR DESCRIPTION
**Pull request by mdeuser**
Update backend actions to add support for the "V2" API GW interface
- New package for actions configured without whisk authentication annotation
- Updated action tests
- Ansible updates to add a new V2 API GW host property

Partial replacement for PR #2031.  A separate PR for only CLI changes will complete the replacement.

### CODE REVIEW

**Comment by mdeuser** [comment]
PG2 1296

**Comment by mdeuser** [comment]
@csantanapr - would you please review the javascript actions
@markusthoemmes - would you mind confirming the ansible changes

Thanks!

File: ansible/roles/routemgmt/files/installRouteMgmt.sh [hunk 1]
```diff
@@ -61,3 +63,27 @@ $WSK_CLI -i --apihost "$APIHOST" action update --auth "$AUTH" "$NAMESPACE/routem
 -a description 'Delete the API' \
 --kind nodejs:default \
 -a web-export true -a final true -a require-whisk-auth true
+
+echo Installing apimgmt package
+$WSK_CLI -i --apihost "$APIHOST" package update --auth "$AUTH"  --shared no "$NAMESPACE/apimgmt" \
+-a description "This package manages the gateway API configuration." \
+-p gwUser "$GW_USER" \
+-p gwPwd "$GW_PWD" \
+-p gwUrl "$GW_HOST" \
```
Comment (mdeuser) [inline ansible/roles/routemgmt/files/installRouteMgmt.sh]: @csantanapr - true, when we completely remove the v1 interface, the `gwUrl` will no longer be needed.  right now, the separate `apimgmt` and `routemgmt` packages leverage the exact same code which requires the `gwUrl` value.  imho, i'd like to handle the v1 removal in a separate pr.

**Comment by mdeuser** [comment]
@markusthoemmes - I believe you've already looked at a sister PR.  The Ansible related updates are all contained in
https://github.com/openwhisk/openwhisk/pull/2067/commits/cd88349c99f84c6bed2b54e26d340d32c5b6edfc

**Comment by cbickel** [comment]
Ansible changes look good to me.

**Comment by mdeuser** [comment]
PG3 502

**Pull request merged**